src/morpho-chainlink/interfaces/IMorphoChainlinkOracleV2Factory.sol

```diff
@@ -21,11 +21,10 @@ interface IMorphoChainlinkOracleV2Factory {
     /// @dev Here is the list of assumptions that guarantees the oracle behaves as expected:
     /// - Feeds are either Chainlink-compliant or the address zero.
     /// - Feeds have the same behavioral assumptions as Chainlink's.
-    /// - Feeds are set in the correct order.
```

Comment:
Should we do the same thing for feeds ? Ie:
`The feeds, if set, are Chainlink compliant.`
or actually:
`The feeds, if set, are Chainlink-interface compliant.`